Sources/Commands/SwiftTool.swift

```diff
@@ -499,43 +520,43 @@ public class SwiftTool {
                 throw ExitCode.failure
             } else {
                 diagnostics.emit(warning: "Did not find optional .netrc file at \(resolvedPath.pathString).")
-                return nil
+                return .none
```

Comment:
I've noticed this change a couple of diffs.  Is this the recommended style for conditionals?

Reply:
I prefer to use `.none` when the optional marks a clear functional decision, e.g. `let cache = .none` is more semantic in my mind then `let cache = nil`

Reply:
Agreed.  I'll start doing the same, and if/when we do a swift-format pass on the whole codebase we can do the rest then.